src/index.html

```diff
@@ -7,8 +7,65 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#home"><img src="images/logo.png" alt="MOYO logo"></a>
+      <nav>
+        <ul class="nav__list">
+          <li class="list__item">
+            <a
+            class="list__link is-active"
+            href="#apple.com">Apple
+            </a>
```

Comment:
[HTML formatting](https://mate-academy.github.io/style-guides/htmlcss.html#html-formatting) Get used to it😉